python/README.md

```diff
@@ -2,7 +2,7 @@
 These steps helps you configure your AWS credentials into the requiered files and initialize terraform backends.
 
 1. Change directory to the root of your git repo
-2. Run the following command `docker run -it -v ${PWD}:/terraform wizelineacademy/wz_terraform:latest`
+2. Run the following command `docker run -it -v ${PWD}/..:/terraform wizelineacademy/wz_terraform:latest`
```

Comment:
Since step 1 mentions we should be in the root directory of the repository, do we still need the `${PWD}/..`?